### ISSUE TITLE
[Dataset request]: discrepencies_and_error_metrics_npj2023

### ISSUE THREAD

**Issue by jvita** [colabfit/data-lake#21]
### Name

Josh Vita

### Email

vita1@llnl.gov

### Dataset name

discrepencies_and_error_metrics_npj2023

### Authors

Yunsheng Liu, Xingfeng He, Yifei Mo

### Links

* Paper: https://www.nature.com/articles/s41524-023-01123-3#data-availability
* Data: https://github.com/mogroupumd/Silicon_MLIP_datasets

### Dataset description

This repository includes the original mlearn_Si_train dataset, modified with the purpose of developing models with better diffusivity scores by replacing ~54% of the data with structures containing migrating interstitials. The enhanced validation set contains 50 total structures, consisting of 20 structures randomly selected from the 120 replaced structures of the original training dataset, 11 snapshots with vacancy RE from AIMD simulations, and 19 snapshots with interstitial RE from AIMD simulations. We also construct interstitial-RE and vacancy-RE testing sets, each consisting of 100 snapshots of atomic configurations with a single migrating vacancy or interstitial, respectively, from AIMD simulations at 1230 K.

### File details

JSON files, with format documented in git repo

### Method

DFT

### Software

VASP

### Software version(s)

5.4.4

### Additional details

PBE-GGA

### Property types

Atomic forces, Potential energy

### Elements

Si

### Permissions

- [ ] I confirm that I have the necessary permissions to submit this dataset

**Comment by gpwolfe** [comment]
staged for ingest after next database update

**Comment by gpwolfe** [comment]
new database now live
